### PR TITLE
Troubleshooting moved to the installation page

### PR DESCRIPTION
**Pull request by mik-laj**
Recently, @mschickensoup  and [my friend Hektor](https://www.instagram.com/hrabia.hektor/) had this problem. Unfortunately, they couldn't easily find this section in the documentation.

After this change, this page still wouldn't answer all of their questions, but it is possible we could go one step further. Hopefully, I can reserve some time to describe the installation process in more detail one day. Installing Airflow is a problem for many people, so it would be worthwhile to describe it in detail.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -157,3 +157,32 @@ Docker image
...
+  # Note: these instructions are for python3.7 but can be loosely modified for other versions
+  brew install python@3.7
+  virtualenv -p /usr/local/opt/python@3.7/Frameworks/Python.framework/Versions/3.7/bin/python3 .toy-venv
+  source .toy-venv/bin/activate
+  pip install apache-airflow
+  python
+  >>> import setproctitle
+  # Success!
+
+Alternatively, you can download and install Python directly from the `Python website <https://www.python.org/>`__.
```
Comment (mik-laj) [inline docs/installation.rst:188]: Here was the link markdown. Now there is an RST link.

File: docs/installation.rst [hunk 2]
```diff
@@ -157,3 +157,32 @@ Docker image
...
+
+Troubleshooting
+'''''''''''''''
+
+This section describes how to troubleshoot installation issues.
+
+``Symbol not found: _Py_GetArgcArgv``
+"""""""""""""""""""""""""""""""""""""
+
+If you see ``Symbol not found: _Py_GetArgcArgv`` while starting or importing Airflow, this may mean that you are using an incompatible version of Python.
```
Comment (mik-laj) [inline docs/installation.rst:169]: I edited the first two sentences slightly to match the section title.

**Comment by XD-DENG** [comment]
Does it suffice to have a paragraph redirecting users to the corresponding sections of FAQ?

**Comment by mik-laj** [comment]
@XD-DENG How will this benefit us? I think this will only make it harder to find this section.

At one of the [docs meetings](https://github.com/apache/airflow/discussions/12112), the FAQ section was discussed and eventually such a sentence appeared in the meeting notes.

> FAQ content should be moved to where it belongs.

**Comment by XD-DENG** [comment]
> @XD-DENG How will this benefit us? I think this will only make it harder to find this section.
> 
> At one of the [docs meetings](https://github.com/apache/airflow/discussions/12112), the FAQ section was discussed and eventually such a sentence appeared in the meeting notes.
> 
> > FAQ content should be moved to where it belongs.

Just an option to suggest. No strong opinion on this. Feel free to proceed the change of course.

**Review by turbaszek — APPROVED** [review]

**Pull request merged**